test/built-ins/Promise/race/length.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2021 Sergey Rubanov. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 25.4.4.3
```

Comment:
use `esid: sec-promise.race` instead. Same for the other files in this commit with es6id. Most of these are no longer a true reference to es6.